extensions/wikia/AdEngine/js/AdSlotTracker.js

```diff
@@ -0,0 +1,47 @@
+/*global define*/
+define('ext.wikia.adEngine.adSlotTracker',  [
```

Comment:
Could you please define this module in some subdirectory (new one?) and name file with lowercase?